.github/workflows/php-hook-documentation.yml

```diff
@@ -0,0 +1,58 @@
+name: PHP Hook Documentation Generator
+
+on:
+  push:
+    branches:
+      - "release/**"
+    paths:
+      - "**.php"
+      - .github/workflows/php-hook-documentation.yml
+  pull_request:
+    types:
+      - opened
+    branches:
+      - "release/**"
+    paths:
+      - "**.php"
+      - .github/workflows/php-hook-documentation.yml
```

Comment:
Under the current release process, this event will not trigger this workflow because GitHub Actions is intentionally designed to prevent workflows from being chain-triggered by a bot account.

For example, in https://github.com/woocommerce/google-listings-and-ads/pull/2067, the same workflow in that repo was not triggered after `github-actions` bot opened that PR, and then I triggered it manually: https://github.com/woocommerce/google-listings-and-ads/actions/runs/5934182562.

A possible trick is to use the [`workflow_run` event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run). Another way is to remove this block and just rely on the flow of *woorelease* tool because it includes a step to bump product version in PHP file(s) so that this workflow will be triggered by `push` event eventually.